data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1615 @@
+					goto cutpower
+				`	"Should we be worried?"`
+			`	Gavriil gestures neutrally. "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." They look somber, but then perk up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." Their gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
+			label cutpower
+			`	Gavriil gestures at the control panel in front of them. "I do not know for sure, but based on events, I suspect that if we manually induce a power failure we should be able to reverse the process that transported us to this realm." They smile eagerly. "I have everything all set up, and we have people waiting across the station." They look confident. "We think that we can safely spin down the reactor and cut power across a number of these arrays in a controlled fashion. This should replicate the original power failure, except with fewer risks and less damage to the system."`
+			`	They leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." They smile in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown."`
+			`	They hand you a data crystal, and then a large case. "Here is an encrypted data crystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`
+			`	They step back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`
+
+
+
+mission "Remnant: Cognizance 36"
+	name "De-activate Ssil Vida Power"
+	description "Accept this to signal everyone to cut the power."
+	job
+	repeat
+	source "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 35: done"
+		has "Remnant: Ssil Vida Active"
+	on accept
+		event "remnant: ssil vida deactivation"
+		conversation
+			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
+	on enter Postverta
+		fail
+
+
+
+mission "Remnant: Cognizance 37"
```

Comment:
Believe I ran into #1938 here. Not much to be done about it, though. You had the same problem?

Reply:
Yep. The mission won't register as being failed until the next time they land, but that's alright. Nothing depends on it being failed or completed directly, that won't be satisfied soon enough the way things are handled now. Nothing will break if/when that is ever fixed, either.